src/main/resources/data/minecolonies/loot_tables/miner/lucky_ore.json

```diff
@@ -0,0 +1,50 @@
+{
+	"pools": [
```

Comment:
Since you added the buildinglevel condition, I suggest we employ it and limit some of the ores, similar to how the miners y level are designed:
Level1: coal, copper
Level2: iron, lapis
Level3: gold, redstone
Level4: diamond

Reply:
agreed

Reply:
This should be datagenned.  (And should be 5 separate files, as noted above.)

Reply:
Made using datagen, distributed ores across levels:
Level 1: Coal / copper
Level 2: Coal / copper / iron / gold
Level 3: Coal / copper / iron / gold / redstone / lapis
Level 4: Coal / copper / iron / gold / redstone / lapis / diamond / emerald
Level 5: Coal / copper / iron / gold / redstone / lapis / diamond / emerald